### DISCUSSION TITLE
Planet owned by player

### DISCUSSION THREAD

**Discussion by GPaso** [endless-sky/endless-sky#9415] [Ideas]
I would like to have a home base planet where ships and outfits could be stored and maybe investing in local material production that you can sell to other planets.

I immagine this by either terraforming or buying a planet or a moon or even a space station (maybe refurbishing one of those uninhabited ones).

This could even expand the gameplay by creating a player lead faction where you can enroll humans or even aliens to work for you producing materials enabling trade routes with other factions and play even a diplomatic game.

This idea could be entirely out of scope of the gameplay and means a really big developement. I just wanted to share it!

**Comment by lumbar527** [comment]
Yeah, that feels like something that doesn't really belong in Endless Sky, at least the base game. However, that could make an interesting plugin.

**Comment by Saugia** [comment]
We have several plug-ins that add this capability to the game (forgive me but I don't recall individual names), although yes it requires a bit of junkiness and workaround in order to work with our engine (through missions and events).

As for part of vanilla, this would be out of scope of Endless Sky, yes, and is more in-line with other types of "infrastructure" games.

**Reply by lumbar527** [reply]
Here's one example, I think: https://github.com/Hecter94/EndlessSky-PluginArchive/tree/main/Working/Businessman%20Mod/

**Comment by GPaso** [comment]
Thanks! I'll check that out!
